diff --git a/tests/lib/python/account-manager.py b/tests/lib/python/account-manager.py
index 0f1fd9f..899a02d 100755
--- a/tests/lib/python/account-manager.py
+++ b/tests/lib/python/account-manager.py
@@ -9,7 +9,7 @@ import dbus
 from dbus.bus import NAME_FLAG_DO_NOT_QUEUE, REQUEST_NAME_REPLY_EXISTS
 from dbus.mainloop.glib import DBusGMainLoop
 from dbus.service import Object, method, signal
-from gobject import MainLoop
+from gi.repository.GObject import MainLoop
 
 TP = 'org.freedesktop.Telepathy'
 
